javagen/src/main/java/com/azure/autorest/mapper/PomMapper.java

```diff
@@ -53,6 +53,10 @@ private Pom createAzurePom(Project project) {
                 Project.Dependency.JUNIT_JUPITER_ENGINE, true);
         addDependencyIdentifier(dependencyIdentifiers, addedDependencyPrefixes,
                 Project.Dependency.MOCKITO_CORE, true);
+        addDependencyIdentifier(dependencyIdentifiers, addedDependencyPrefixes,
+                Project.Dependency.BYTE_BUDDY, true);
+        addDependencyIdentifier(dependencyIdentifiers, addedDependencyPrefixes,
+                Project.Dependency.BYTE_BUDDY_AGENT, true);
```

Comment:
I actually didn't remember why we add `mockito-core` by default to DPG (under `generate-tests` option)

Reply:
Emm, actually those in DPG is not under `generate-tests` option.

Reply:
@alzimmermsft @srnagar 
Any preference about whether we should include all the test dependencies when generating pom.xml for DPG?

Some like `azure-core-test` and `azure-identity` should actually be almost always needed by dev in follow up work, when adding test cases (also when we generate test cases under `generate-tests` option).

But dev may not use `mockito` as it would only for mock test, while mostly they do record/playback test.

Should I remove `mockito` in DPG pom?

Reply:
If the generated code isn't using Mockito I'd remove it. While Mockito helps out a lot in testing it's one of those dependencies that has a lot of gotchas around support in newer Java versions or gotchas during usage which could be replaced by slightly more verbose testing code (creating mock classes locally, re-writing tests to not use mocking).

Reply:
Got you. Let me remove it from DPG.